src/commands/Games/trivia.ts

```diff
@@ -0,0 +1,88 @@
+import { SkyraCommand, SkyraCommandOptions } from '@lib/structures/SkyraCommand';
+import { ApplyOptions } from '@skyra/decorators';
+import { KlasaMessage, KlasaUser } from 'klasa';
+import TriviaManager, { CATEGORIES, QuestionType, QuestionDifficulty, QuestionData } from '@utils/Games/TriviaManager';
+import { decode } from 'he';
+import { shuffle } from '@utils/util';
+import { MessageEmbed, MessageCollector } from 'discord.js';
+
+@ApplyOptions<SkyraCommandOptions>({
+	cooldown: 5,
+	description: language => language.tget('COMMAND_TRIVIA_DESCRIPTION'),
+	extendedHelp: language => language.tget('COMMAND_TRIVIA_EXTENDED'),
+	usage: '(category:category) [boolean|multiple] [easy|hard|medium] [duration:int{30,60}]',
+	usageDelim: ' '
+})
+export default class extends SkyraCommand {
+
+	// eslint-disable-next-line @typescript-eslint/explicit-member-accessibility
+	#channels = new Set<string>();
+
+	public async run(message: KlasaMessage, [category, questionType = undefined, difficulty = undefined, duration = 30]: [number, QuestionType?, QuestionDifficulty?, number?]) {
+		if (this.#channels.has(message.channel.id)) throw message.language.tget('COMMAND_TRIVIA_ACTIVE_GAME');
+
+		this.#channels.add(message.channel.id);
+		await message.sendLocale('SYSTEM_LOADING');
+		const data = await TriviaManager.getQuestion(category, difficulty, questionType);
+		const possibleAnswers = questionType === QuestionType.BOOLEAN
+			? ['True', 'False']
```

Comment:
Move to i18n, as well as make them lower-case so we can match them with `.toLowerCase()`.

Reply:
I don't think we should do this: the API doesn't have i18n support so it will always return True or False. Plus multiple-choice questions will always be in English (for aforementioned reasons), so I'd prefer boolean questions be in English, for consistency.